.github/workflows/ci.yml

```diff
@@ -17,8 +17,52 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  setup:
+    runs-on: ubuntu-22.04
+    outputs:
+      be-tests: ${{ steps.packages.outputs.be-tests }}
+      pw-tests: ${{ steps.packages.outputs.pw-tests }}
```

Comment:
When I read `pw-tests` I think "password" not "Playwright". We have an outstanding `@todo` to rename that test directory anyway, as it's not semantically descriptive (e.g. "FE integration tests"). But we can address this in follow-up (remembering to change this then, too).

Reply:
I 100% agree